### ISSUE TITLE
Compress fasttext and add to utils

### ISSUE THREAD

**Issue by MateuszOlko** [spacy-pl/utils#30]
DOD:
* scripts to prepare vectors for large and medium model parametrized by the weight of vectors
* dvc pipeline
* add blank model with vectors

**Comment by MateuszOlko** [comment]
I've searched how the vectors look in other model released by explosion. The only language that has large vectors is English, they have around 600K vectors. 
In medium models, explosion team made some trick. "md" models have 20K vectors, but much greater amount of keys (again around 600K). Many keys are linked to the same vector. I think preparing that mapping is out of scope of this task, so I decided to leave it for future work.

SpaCy CLI uses vectors in txt format, while reading vectors to model is easier when its saved in binary. Therefore i'll create both files.

After this research I consider size of vectors as a opinioned decision. Substantiation of my choice will be in notebook.

I propose a new DOD:
* add to dvc prepared fasttext files
* prepare notebook with comparison fasttext vs word to vec and some statistical data
